pkg/volume/portworx/portworx_util.go

```diff
@@ -211,7 +214,11 @@ func (util *portworxVolumeUtil) ResizeVolume(spec *volume.Spec, newSize resource
 	}
 
 	vol := vols[0]
-	newSizeInBytes := uint64(volumehelpers.RoundUpToGiB(newSize) * volumehelpers.GiB)
+	requestGiB, err := volumehelpers.RoundUpToGiB(newSize)
+	if err != nil {
+		return err
+	}
+	newSizeInBytes := uint64(requestGiB * volumehelpers.GiB)
```

Comment:
could requestGiB * volumehelpers.GiB overflow?

Reply:
I suppose here it just needs to do something like 

newSizeInBytes, ok := newSize.AsUint64

assuming  AsUint64 function exists, if not then uint64(newSize.AsInt64)

Reply:
yeah that is what I ended up doing.